lib/ansible/modules/packaging/os/zypper_repository.py

```diff
@@ -212,7 +212,7 @@ def repo_exists(module, repodata, overwrite_multiple):
             if repodata[kw] == oldr[kw] and oldr not in repos:
                 repos.append(oldr)
 
-    if len(repos) == 0:
+    if not repos:
```

Comment:
I find the old style more readable in the context of the following `elif` statements.